<ui version="4.0" >
 <class>EditExt_fi</class>
 <widget class="QWidget" name="EditExt_fi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frmDecl" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lbGenitive" >
        <property name="text" >
         <string>Genitive:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leGenitive" />
      </item>
      <item>
       <widget class="QLabel" name="lbPartitive" >
        <property name="text" >
         <string>Partitive:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePartitive" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmConj" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lbFirstPersSg" >
        <property name="text" >
         <string>min√§</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFirstPersSg" />
      </item>
      <item>
       <widget class="QComboBox" name="cboxConjClass" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
